ma_statement.h

```diff
@@ -324,8 +336,6 @@ void         MADB_CspsFreeDAE(MADB_Stmt *Stmt);
     " WHEN 'mediumint' THEN 0" \
     " WHEN 'int' THEN 0" \
     " WHEN 'bigint' THEN 0" \
-    " WHEN 'double' THEN IF(NUMERIC_SCALE IS NULL, 6, NUMERIC_SCALE)" \
-    " WHEN 'float' THEN IF(NUMERIC_SCALE IS NULL, 0, NUMERIC_SCALE)" \
```

Comment:
Don't we want this query to handle double/float?

Reply:
If I understand correctly https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/decimal-digits?view=sql-server-ver16, for double/float ( approximate data types) decimal digits is undefined.